### DISCUSSION TITLE
any way to randomly select from a preset folder of poses instead of run the whole batch?

### DISCUSSION THREAD

**Discussion by Sacred-Reputation** [adieyal/sd-dynamic-prompts#569] [Q&A]
i have lots of different poses in a folder, and lots of different outfits for my character in a "Dynamic Prompt" wildcard - i'd like to run a different combination of pose & outfit each generation.

right now seems like i can only run the whole batch, or manually select which pose i want.

integrate poses as dynamic prompt wildcards?

**Comment by adieyal** [comment]
I'm not sure I follow

What do you want to be different that this prompt doesn't do for you?
`A man in a __pose__ pose wearing __outfit__`

**Comment by Sacred-Reputation** [comment]
I have a folder of .png files - lineart poses intended for ControlNet - I'd like it to choose ONE of those files at a time, at random, for each generation

not text descriptions of poses, actual .png files
